### PR TITLE
Add Attributes

### PR DESCRIPTION
**Pull request by Bandov**
Skeleton code to be used for implementation of v1.2

### CODE REVIEW

**Comment by ReflectiveObsidian** [comment]
Required for issue #8

File: src/main/java/seedu/address/model/person/AgeAttribute.java
Comment (ReflectiveObsidian) [inline src/main/java/seedu/address/model/person/AgeAttribute.java:1]: Is AgeAttribute necessary, given that BirthdayAttribute exists? Age can be dynamically calculated from the brithday, and does not need to be periodically updated when done in that manner.

File: src/main/java/seedu/address/model/person/GenderAttribute.java
Comment (ReflectiveObsidian) [inline src/main/java/seedu/address/model/person/GenderAttribute.java:1]: Would this binary attribute be more accurately named SexAttribute? (See https://cihr-irsc.gc.ca/e/48642.html)

**Review by ReflectiveObsidian — COMMENTED** [review]
Some comments

File: src/main/java/seedu/address/model/person/AgeAttribute.java
Comment (Bandov) [reply src/main/java/seedu/address/model/person/AgeAttribute.java:1]: ah ok i get it im removing age attribute then

File: src/main/java/seedu/address/model/person/GenderAttribute.java
Comment (Bandov) [reply src/main/java/seedu/address/model/person/GenderAttribute.java:1]: updated attribute and test cases

**Pull request closed without merging**